src/apps/Permissions/AssignPermissionPanel.js

```diff
@@ -196,6 +197,18 @@ class AssignPermissionPanel extends React.PureComponent {
             {'Add permission'}
           </Button>
 
+          {this.state.assignEntityAddress === ANY_ENTITY && (
```

Comment:
Note that we should actually use [`isAnyEntity()`](https://github.com/aragon/aragon/blob/d8f0fe556e2c2986250768cf0b8f26d8030f1b06/src/permissions.js#L34) instead—doing this check right now is generally not robust due to checksumming concerns.

Reply:
True!